wled00/FX_fcn.cpp

```diff
@@ -455,9 +455,9 @@ CRGBPalette16 IRAM_ATTR &Segment::currentPalette(CRGBPalette16 &targetPalette, u
 // relies on WS2812FX::service() to call it for each frame
 void Segment::handleRandomPalette() {
   // is it time to generate a new palette?
-  if ((millis()/1000U) - _lastPaletteChange > randomPaletteChangeTime) {
+  if (((millis()/1000U) & 0xFFFF) - _lastPaletteChange > randomPaletteChangeTime) {
```

Comment:
the explicit cast to `uint16_t` would have achieved the same, but would be more understandable.